doc/lsst.ctrl.bps.htcondor/userguide.rst

```diff
@@ -159,6 +159,22 @@ initially run with 2 GB of memory and failed because of exceeding the limit,
 result the entire workflow fails again due to other reasons, the job will ask
 for 2 GB of memory during the first execution after the workflow is restarted.
 
+If you did not set the ``memoryMultiplier`` option, this command:
+
+.. code-block:: sh
+
+   condor_q -hold $USER
+
+will show any held jobs and the reasons for their being held, including running out of memory.
+
+This command:
+
+.. code-block:: sh
+
+   condor_q $USER | awk '{if($6 == "H") print "condor_qedit",$1,"RequestMemory 4096; condor_release",$1}' | bash
```

Comment:
This assumes all of the user's held jobs are from runs submitted without automatic tries and are held because of memory issues.  But based on questions I've seen on slack this is more than likely the case.  We can update this later to be more complicated but more accurate if actually need it.